ext/dev.js

```diff
@@ -117,10 +117,14 @@
 			}
 		},
 		_logger:function(type, arr){
-			if(console.log.apply){
-				console[type].apply(console, arr);
-			} else {
+			if (typeof console.log == "object") {
```

Comment:
Ok, I researched this a bit myself and it seems that console.log is an object but you can call it like a function. Really bizarre.  However I also found out that `console` doesn't exist until you open the dev tools. So we should really be checking typeof console as well.